src/CodeEditor/driver.js

```diff
@@ -132,4 +132,23 @@ export default class CodeEditorDriver
         this.wrapper.simulate( 'mouseleave' );
         return this;
     }
+
+    pressKey( keyCode )
```

Comment:
Function should be named `keyPress` as it corresponds to the `onKeyPress` callback prop:
```suggestion
    keyPress( keyCode )
```